batch/test/test_batch.py

```diff
@@ -214,13 +183,9 @@ def test_batch(self):
         j2 = b.create_job('alpine', ['sleep', '1'])
         j3 = b.create_job('alpine', ['sleep', '30'])
 
-        # test list_jobs
-        jobs = self.batch.list_jobs()
-        self.assertTrue(
-            set([j.id for j in jobs]).issuperset([j1.id, j2.id, j3.id]))
-
+        j1.wait()
```

Comment:
Shouldn’t all calls to the batch raise if the batch is cancelled?

Reply:
The batch is cancelled later. I don't see the issue.